test/run.py

```diff
@@ -38,7 +38,14 @@
 import shutil
 from unittest import TestLoader, main
 
-from vsc.utils.run import CmdList, run, run_simple, run_asyncloop, run_timeout, RunQA, RunTimeout
+#import logging
+#logging.basicConfig(level=logging.DEBUG)
```

Comment:
@stdweird Remove these, since they're not used?

Reply:
i can't enable them by default, it screws up some of the tests; but i'll uncomment the import logging :wink: